streams/src/test/java/org/apache/kafka/streams/processor/TopologyBuilderTest.java

```diff
@@ -603,27 +602,25 @@ public void shouldThroughOnUnassignedStateStoreAccess() throws Exception {
         config.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getAbsolutePath());
         final StreamsConfig streamsConfig = new StreamsConfig(config);
 
-        try {
-            final TopologyBuilder builder = new TopologyBuilder();
-            builder
-                .addSource(sourceNodeName, "topic")
-                .addProcessor(goodNodeName, new LocalMockProcessorSupplier(), sourceNodeName)
+        final TopologyBuilder builder = new TopologyBuilder();
+        builder.addSource(sourceNodeName, "topic")
+                .addProcessor(goodNodeName, new LocalMockProcessorSupplier(),
+                        sourceNodeName)
                 .addStateStore(
-                    Stores.create(LocalMockProcessorSupplier.STORE_NAME).withStringKeys().withStringValues().inMemory().build(),
-                    goodNodeName)
-                .addProcessor(badNodeName, new LocalMockProcessorSupplier(), sourceNodeName);
+                        Stores.create(LocalMockProcessorSupplier.STORE_NAME)
+                                .withStringKeys().withStringValues().inMemory()
+                                .build(), goodNodeName)
+                .addProcessor(badNodeName, new LocalMockProcessorSupplier(),
+                        sourceNodeName);
 
+        try {
             final ProcessorTopologyTestDriver driver = new ProcessorTopologyTestDriver(streamsConfig, builder.internalTopologyBuilder);
```

Comment:
We should improve this test by moving this line outside/before `try`

Reply:
Moving outside the `try` results in the test failing because this line is throwing the exception

  org.apache.kafka.streams.errors.StreamsException: failed to initialize processor badGuy

Reply:
I see. Than please remove the next line `driver.process` and simplify to
```
new ProcessorTopology(...)
```

(we don't need variable `driver` for this case)